Sandbox/Sandbox/AppDelegate.swift

```diff
@@ -52,7 +52,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             print("addPasswordlessCallback \(result)")
         }
         
-        return true
+        return reachfive.application(application, didFinishLaunchingWithOptions: launchOptions)
```

Comment:
For @talking-tortoise : Integrator must now call this method at app initialisation.

Reply:
Cheers @FrancoisDevemy5 -- will you open a DOC ticket with this so I make sure I follow up on it. 🍻 🙏🏼